index.js

```diff
@@ -15,13 +15,13 @@ class Client {
       throw new TypeError('Please provide auth options.')
     }
 
-    const { email, ethAddress, apiKey, password } = auth
+    const { email, ethAddress, apiKey, password, userId } = auth
 
-    if (!email && !ethAddress && !apiKey) {
+    if (!email && !ethAddress && !apiKey && !userId) {
```

Comment:
Same as above, I think it is ok to not pass `userId`, in that case we will later initialize it to the trial user ID.